apps/dashboard/src/modules/auth/views/AuthLocalView.vue

```diff
@@ -70,7 +70,7 @@ import { toTypedSchema } from '@vee-validate/yup';
 import * as yup from 'yup';
 import InputText from 'primevue/inputtext';
 import { useToast } from 'primevue/usetoast';
-import { AxiosError } from 'axios';
+import type { AxiosError } from 'axios';
```

Comment:
Outside the scope of this PR

Reply:
yeah I thought yarn 4 introduced this a problem, because the builds failed, but I ended up fixing it by retrying it five times and it working. So should be not needed anymore indeed